app/routes/errors/HTTPError.js

```diff
@@ -6,9 +6,9 @@ import { Link } from 'react-router';
 import renderAbakus from './renderAbakus';
 
 const HTTPMapping = {
-  '400': '400 Bad Method',
-  '401': '401 Unauthorized',
-  '403': '403 Forbidden',
+  '400': 'Noe gikk veldig galt, Webkom er på saken!',
```

Comment:
400 isn't that bad :) it _should_ mean that the user did something wrong, and not us. i'm not sure where this would pop up, but maybe something like `Noe gikk galt med forespørselen`?